spec.html

```diff
@@ -72,6 +72,11 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
         1. ReturnIfAbrupt(_next_).
         1. If _next_ is *false*, then
           1. Set _iteratorRecord_.[[Done]] to *true*.
+          1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
+          1. If _remainingElementsCount_.[[Value]] is 0, then
+            1. Let _error_ be a newly created `AggregateError` object.
+            1. Perform ! CreateDataProperty ( _error_, `"errors"`, _errorsArray_ ).
+            1. Return ThrowCompletion ( _error_ ).
```

Comment:
Need to do this at [the other reject branch](https://github.com/tc39/proposal-promise-any/blob/1f58bffabf3f293fdc9ad82d096e6f336c29aa60/spec.html#L115-L116) too.

Reply:
Oh, yes sure